test_main.py

```diff
@@ -3,97 +3,44 @@
 # To run the tests, use: python3 -m pytest --capture=sys
 
 from database import Database
+from filters import Filter
+import filters
 from main import main
 from test_utils import region_dict
+import random
 
 db = Database()
 print(len(db))
 
 
-def test_no_args(capsys):
-    main([__file__])
-    out, err = capsys.readouterr()
-    assert out.startswith("No command line arguments specified.")
-
-
-def test_three_args(capsys):
```

Comment:
Now with filter mixing the new command engine accepts way more then 3 arguments.